### PR TITLE
Add manage_request functionality

### PR DESCRIPTION
**Pull request by ManasaRaoK**
Add manage_request functionality that includes the following:
a. POST on /manageRequest endpoint
b. GET on /manageRequest/jobs/<job_id>

### CODE REVIEW

**Comment by ManasaRaoK** [comment]
@AndreyMenezes , @rodneyhbrown7 , @walteraa , @skovic - Can you please review

**Comment by ManasaRaoK** [comment]
The tests are now added

**Comment by ManasaRaoK** [comment]
@AndreyMenezes @rodneyhbrown7 @walteraa @skovic , if there are no further review comments, can this PR be merged

File: lib/xclarity_client/manage_request_management.rb [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+            discovery_ip_addr = val[0]["ipAddresses"][0]
+            deploy_hash.merge!({:ipAddresses => [discovery_ip_addr]})
+
+            if val[0]["type"].eql? "Rackswitch"
+              deploy_hash.merge!({:os => val[0]["os"]})
+            end
+          end
+        end
+      end
+      if force.eql? "true" or force.eql? "True"
```
Comment (AndreyMenezes) [inline lib/xclarity_client/manage_request_management.rb]: I already talk about this statement.

Comment (AndreyMenezes) [inline lib/xclarity_client/manage_request_management.rb]: I already talk about this statement.

**Pull request closed without merging**